views/pages/admin/documents.pug

```diff
@@ -1 +1,20 @@
-extends /views/templates/admin-base.pug
\ No newline at end of file
+extends /views/templates/admin-base.pug
+
+append styles
+  link(href='/assets/css/pages/admin/documents.css' rel="stylesheet")
+
+append scripts
+  script(src='/assets/js/admin/documents.js' defer)
+
+block body 
+  +section()
+    h2 Upload new
+
+    input(type="file" name="documents" required multiple accept=".pdf")
+    button(id="upload-btn" type="submit") Upload documents
+  +section("white", "documents-section")
+    h1 Documents
+    div(id="document-list")
+ 
+  +section() 
+    h2 Manage existing
```

Comment:
should we keep this section? what's the plan w it?

Reply:
removed it for now